src/components/grid-aware/VideoHeader/VideoHeader.stories.js

```diff
@@ -28,5 +28,4 @@ DefaultVideoHeader.args = {
     { text: "Donate", internalLink: "/new/donate" },
     { text: "Volunteer", internalLink: "/new/volunteer" },
```

Comment:
I understand this is the `stories.js` file and won't be used in production but did we want to update the `internalLink` paths to `"/donate"` and `"/volunteer"` respectively even if it doesn't make a difference?

Reply:
I don't think it's really necessary, and if we were to do something here, I'd probably want to just replace these with completely dummied out URLs, just to emphasize that the actual URLs in the links is not important in the Storybook story.

Reply:
Okay! Thanks for the reference.